### PR TITLE
Add Cloud Function that deletes VM instances created without CMEK.

### PR DESCRIPTION
**Pull request by zachberger**
This Cloud Function detects newly created VM Instances that have disks attached
with a Google-managed encryption key attached and then immediately deletes
any such VM instances.

### CODE REVIEW

**Comment by zachberger** [comment]
> could the example be extended with a noncompliant VM to prove the functionality without requiring the end users to add additional resources?

This sounds frustrating. Typically when I grab an example or tool I want it to work with as little configuration as possible. If I do this the end user has to cleanup the TF and remove this resource and any other dependencies.

**Comment by zachberger** [comment]
@aaron-lane I don't have the ability to assign this back to you

**Comment by morgante** [comment]
> This sounds frustrating. Typically when I grab an example or tool I want it to work with as little configuration as possible. If I do this the end user has to cleanup the TF and remove this resource and any other dependencies.

Agreed.

**Comment by aaron-lane** [comment]
@zachberger Typically, examples should be as self-contained as possible. If this example has practical applications outside of demonstrating usage of the module then we should add it as a submodule so that it can be properly supported.

**Comment by zachberger** [comment]
This is exhausting. Ive been sent from repo to repo being told to contribute in different ways. At this point I don't even want to contribute this anymore.

**Comment by morgante** [comment]
I'm reopening this because I concur with @zachberger that having an entire VM instance be part of the config just for a simple example is extraneous and unnecessary. As far as I am concerned, this PR is fine to merge as-is.

File: examples/delete-vms-without-cmek/function_source/cmek.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+func init() {
+	var err error
+
+	computeService, err = compute.NewService(context.Background())
+	if err != nil {
+		log.Fatalf("Could not create compute service: %v\n", err)
+	}
+}
+
+func RecieveMessage(ctx context.Context, msg *pubsub.Message) error {
```
Comment (morgante) [inline examples/delete-vms-without-cmek/function_source/cmek.go]: nit. spelling (ReceiveMessage)

**Pull request merged**